src/library/Graphs/PayoutLine.tsx

```diff
@@ -37,30 +37,33 @@ ChartJS.register(
 export const PayoutLine = ({
   payouts,
   averageWindowSize,
+  maxPayoutDays,
   height,
   background,
 }: PayoutLineProps) => {
-  const graphablePayouts = payouts.slice(averageWindowSize); // Leave out oldest "averageWindowSize" number of values for the average window
+  const graphablePayouts = payouts.slice(-maxPayoutDays); // Take only values included in latest maxPayoutDays.
```

Comment:
I think that after the change the code is self-explanatory - no need for the comment.